docs/src/components/DocLayout/index.tsx

```diff
@@ -71,120 +73,131 @@ export default function DocLayout({
-                  onCollapse={() => {
-                    // hack for when collapsing an overflowing DocNavigationItem in Modal
-                    // causes the fixed ModalFooter to be stuck at the bottom of the screen
-                    setTimeout(() => {
-                      // causing Modal to reposition ModalFooter
-                      window.dispatchEvent(new Event("resize"));
-                    }, 70); // not sure why, but this delay is necessary (the exact threshold is 62)
-                  }}
-                />
-              }
-            />
-            <StyledMiddle>
-              <Stack
-                flex
-                direction="column"
-                desktop={{ direction: "row", spacing: "large", align: "stretch" }}
-              >
-                {trail && (
-                  <Hide block on={["smallMobile", "mediumMobile", "largeMobile", "tablet"]}>
-                    <StyledDocNavigationWidth>
-                      <StyledDocNavigationWrapper>
-                        <DocNavigation currentUrl={path} />
-                      </StyledDocNavigationWrapper>
-                    </StyledDocNavigationWidth>
-                  </Hide>
-                )}
-                <StyledMain>
-                  {trail && <Breadcrumbs trail={trail} />}
-                  <Box padding={{ bottom: "XLarge" }}>
-                    <Stack inline align="center" spaceAfter="small">
-                      <AddBookmark />
-                      <Heading as="h1" type="display">
-                        {title}
-                      </Heading>
-                    </Stack>
-                    {description && (
-                      <Box padding={{ left: "XXLarge" }}>
-                        <Text>{description}</Text>
-                      </Box>
-                    )}
-                  </Box>
-                  <StyledMobileOutdent>
-                    {(tabs || headerLink) && (
-                      <Box
-                        display="flex"
-                        align="end"
-                        justify={tabs && tabs.length > 0 ? "between" : "end"}
-                        tablet={{ maxWidth: "80%" }}
-                      >
-                        {tabs && <Tabs activeTab={location.pathname} tabs={tabs} />}
-                        {headerLink && (
-                          // align with tabs
-                          <Box padding={{ bottom: "XXSmall" }}>
-                            <HeaderLink href={headerLink} />
-                          </Box>
-                        )}
-                      </Box>
-                    )}
-                    <Grid columns="1fr" tablet={{ columns: `${tocHasItems ? "80% 20%" : "100%"}` }}>
-                      {tocHasItems && (
-                        <StyledTocWrapper>
-                          <Hide on={["smallMobile", "mediumMobile", "largeMobile"]}>{Toc}</Hide>
-                        </StyledTocWrapper>
+    <>
+      <Helmet>
```

Comment:
For proper SEO and accessibility, we'll want more items in the helmet (like language for accessibility). It'd make sense to me to make it its own component (then you don't have to repeat the similar code as in Layout, and won't leave things out in one place but not the other (like the meta charset, unless there's a reason for that difference?).